### ISSUE TITLE
Consider downloading in parallel

### ISSUE THREAD

**Issue by JustinGrote** [PowerShell/PSResourceGet#98]
https://github.com/PowerShell/PowerShellGet/blob/98aadd21b741c69ad782c6438663b18a309ff9e3/src/InstallPSResource.cs#L686

You went to all the trouble to implement async, why not download in parallel, wait for the tasks to complete, and then move on, rather than download one package at a time?

**Comment by SydneyhSmith** [comment]
Thanks @JustinGrote we are hoping to achieve this, going to mark as a duplicate of the issue we are tracking for this #69
